recipe/meta.yaml

```diff
@@ -148,7 +148,7 @@ outputs:
         - cross-r-base {{ r_base }}  # [build_platform != target_platform]
         - r-rcppspdlog >=0.0.19      # [build_platform != target_platform]
         - r-matrix                   # [build_platform != target_platform]
-        - r-bit64                    # [build_platform != target_platform]
+        - r-bit64 >=4.6.0.1          # [build_platform != target_platform]
```

Comment:
Due to the quirks of conda and R, this pin only affects the cross-compiled builds. Unfortunately you have to find-replace 3 times (build/host/run)